src/firejail/usage.c

```diff
@@ -208,6 +208,7 @@ static char *usage_str =
 	"\twhitelist the syscalls specified by the command.\n"
 	"    --seccomp.print=name|pid - print the seccomp filter for the sandbox\n"
 	"\tidentified by name or PID.\n"
+	"    --seccomp-error-action=errorno|kill - change error code or kill process.\n"
```

Comment:
There are a few places where "errorno" is used.
I think it's clearer if "errno" would be used instead.

Reply:
OK. Though "errorno" was used elsewhere, it's slightly less technical than Unix term "errno".